Loop/View Models/BolusEntryViewModel.swift

```diff
@@ -585,15 +585,17 @@ final class BolusEntryViewModel: ObservableObject {
                     potentialBolus: enteredBolusDose,
                     potentialCarbEntry: potentialCarbEntry,
                     replacingCarbEntry: originalCarbEntry,
-                    includingPendingInsulin: true
+                    includingPendingInsulin: true,
+                    considerPositiveVelocityAndRC: true
```

Comment:
So, if I understand this correctly, we want the graph to show the results with positive effects, but the recommended bolus to *not* include positive effects (assuming feature flag is false)? And this is to prevent someone from wondering why the recommended bolus isn't greater if the graph shows a final result above target?

Reply:
> So, if I understand this correctly, we want the graph to show the results with positive effects, but the recommended bolus to not include positive effects (assuming feature flag is false)?